utils/atomic_vector.hh

```diff
@@ -22,6 +22,12 @@ class atomic_vector {
     mutable seastar::rwlock _vec_lock;
 
 public:
+    bool empty() const noexcept {
```

Comment:
nit: atomic_vector changes do not fit the patch description

Reply:
dropped in v2 ([06da3f3](https://github.com/scylladb/scylladb/commit/06da3f30f0c2da5e684fbff7b517e97d70d70559))